mofacts/client/views/experiment/card.js

```diff
@@ -2661,20 +2662,21 @@ async function resumeFromComponentState() {
 
   await updateExperimentState(newExperimentState, 'card.resumeFromComponentState');
   
-  //check if user feedback settins are enabled, if so redirect to settings menu
-  await getFeedbackParameters(curTdfUnit);
+
 
   // Notice that no matter what, we log something about condition data
   // ALSO NOTICE that we'll be calling processUserTimesLog after the server
   // returns and we know we've logged what happened
   checkSyllableCacheForCurrentStimFile(processUserTimesLog);
+
+  //check if user feedback settins are enabled, if so redirect to settings menu
+  getFeedbackParameters(curTdfUnit);
```

Comment:
@JRustyHaner Did you test this?

Reply:
Tested on testtdf and a tdf that did not have the allowFeedbackTypeSelect

Reply:
You don't have branch selection here. You need to selectively go down one path or the other